storage_service/locations/tests/test_arkivum.py

```diff
@@ -12,6 +12,11 @@
 ARKIVUM_DIR = os.path.abspath(os.path.join(FIXTURES_DIR, 'arkivum'))
 
 
+def get_pkg_uuid_path(package_uuid):
```

Comment:
Would you mind adding a comment here for the reader with an example? E.g.:
`b9a07040-4f08-4a9c-b88a-5cc325e17e7e => b9a0/7040/4f08/4a9c/b88a/5cc3/25e1/7e7e`